src/tfrnnlmbin/Makefile

```diff
@@ -0,0 +1,36 @@
+# Tis directory is not compiled automatically when you do make under src/
+# to compile the TensorFlow related code, do the following:
+
+# you would need to run tools/extras/install_tensorflow_cc.sh first to generate
+# the static library files required for compilation, and then
+
+# cd ${KALDI_ROOT}/src
+# cd tfrnnlm
+# make
+# cd ../tfrnnlmbin/
+# make
+
+TENSORFLOW = $(shell pwd)/../../tools/tensorflow
+
+all:
+
+EXTRA_CXXFLAGS = -Wno-sign-compare -I$(TENSORFLOW)/bazel-tensorflow/external/protobuf/src -I$(TENSORFLOW)/bazel-genfiles -I$(TENSORFLOW) -I$(TENSORFLOW)/tensorflow/contrib/makefile/downloads/eigen/ -Wl,-rpath=$(shell pwd)/../../tools/tensorflow/bazel-bin/tensorflow/
+include ../kaldi.mk
+
+BINFILES = lattice-lmrescore-tf-rnnlm
+
+OBJFILES =
+
+TESTFILES =
+
+ADDLIBS = ../lat/kaldi-lat.a ../lm/kaldi-lm.a ../fstext/kaldi-fstext.a \
+          ../hmm/kaldi-hmm.a ../tree/kaldi-tree.a ../util/kaldi-util.a \
+          ../matrix/kaldi-matrix.a \
+          ../base/kaldi-base.a ../tfrnnlm/kaldi-tensorflow-rnnlm.a \
+          $(shell pwd)/../../tools/tensorflow/bazel-bin/tensorflow/libtensorflow_cc.so
+
+LDLIBS +=  -lz -ldl -fPIC -lrt
+#LDLIBS += -rpath,$(shell pwd)/../../tools/tensorflow/bazel-bin/tensorflow/
```

Comment:
Try adding it to LDFLAGS, e.g. using
`LDFLAGS += ....`
that's where it belongs.
Break up EXTRA_CXXFLAGS with backslash if needed, I couldn't see that because it was off the screen.
And remove that comment about `LDLIBS += ...`